.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,36 @@
+name: Express Rescue gitHub actions
+
+on: pull_request
+
+jobs:
+  scripts:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+
+      - name: Setting up node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Build the code
+        run: npm run build
+
+      - name: Run lint
+        run: npm run lint
+
+      - name: Run test
+        run: npm run test 
+
+      - name: Run coverage
+        run: npm run coverage
+
+      - name: Run coveralls
+        uses: coverallsapp/github-action@v1.1.0
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          parallel: true
```

Comment:
According to this gha, setting parallel=true requires that we also use set the property `flag-name`

Reply:
Also, it might be a good idea to add one more job just for running the steps `npm run coverage` and the `coveralls action`, cuz we don't need to send the coverage report for all node versions since they'd contain the same report. Sending it reports for the latest node version is enough :)

Reply:
But If this is what that `parallel: true` is meant for, then I guess merging as is should be fine

Reply:
Yeah, my bad. Since we are only using it for the latest version I did another job. Parallel would mean check the coverage of all versions. Already fixed!